_posts/blog/2016-04-15-subscribe_on.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "RxJava: subscribeOn vs observeOn"
+modified:
+categories: blog
+author: tomek_polanski
+excerpt:
+tags: [RxJava]
+image:
+date: 2016-04-14T13:00:55-01:00
+---
+
+
+## RxJava: subscribeOn vs observeOn
+
+One of the strongest aspects of RxJava is the simple way to schedule work on a desired thread using either ``subscribeOn`` or ``observeOn``. Understanding how each one works is crucial.  
```

Comment:
Regarding the first point, I would like to make it clear that I mean thread operations